pkg/cfg/config.go

```diff
@@ -20,26 +20,26 @@ import (
 
 type LookupEnv func(key string) (string, bool)
 
-//go:generate mockery -name Config
+// go:generate mockery -name Config
 type Config interface {
 	AllKeys() []string
 	AllSettings() map[string]interface{}
-	Get(string) interface{}
-	GetBool(string) bool
-	GetDuration(string) time.Duration
-	GetInt(string) int
-	GetIntSlice(string) []int
-	GetFloat64(string) float64
-	GetString(string) string
-	GetStringMap(key string) map[string]interface{}
-	GetStringMapString(string) map[string]string
-	GetStringSlice(string) []string
-	GetTime(key string) time.Time
+	Get(key string, optionalDefault ...interface{}) interface{}
+	GetBool(key string, optionalDefault ...bool) bool
+	GetDuration(key string, optionalDefault ...time.Duration) time.Duration
+	GetInt(key string, optionalDefault ...int) int
+	GetIntSlice(key string, optionalDefault ...[]int) []int
+	GetFloat64(key string, optionalDefault ...float64) float64
+	GetString(key string, optionalDefault ...string) string
+	GetStringMap(key string, optionalDefault ...map[string]interface{}) map[string]interface{}
+	GetStringMapString(key string, optionalDefault ...map[string]string) map[string]string
+	GetStringSlice(key string, optionalDefault ...[]string) []string
+	GetTime(key string, optionalDefault ...time.Time) time.Time
 	IsSet(string) bool
 	UnmarshalKey(key string, val interface{})
 }
 
-//go:generate mockery -name GosoConf
+//  go:generate mockery -name GosoConf
```

Comment:
There shouldn't be any spaces here, right

Reply:
no, i need spaces here, as mockery is broken in this case
had to adjust the mocks manually.. so i don't want them overwritten by generate

Reply:
Hmm, maybe we should put a comment there in that case, otherwise someone might think this is an error and try to "fix" it